tests/datatypes/test_bytes.py

```diff
@@ -216,6 +216,13 @@ def test_index(self):
             print(b'abcd'.find(b'ab', 3, 4))
         """, exits_early=True)
 
+    def test_contains(self):
+        self.assertCodeExecution("""
+            print(b'py' in b'pybee')
+            print(b'bee' in b'pybee')
+            print(b'ybe' in b'pybee')
+            print(b'test' in b'pybee')
```

Comment:
This is a good start!

What about handling cases like:

```
>>> 100 in b'pybee'  # 100 is the character b'd'
False
>>> 101 in b'pybee'  # 101 is the character b'e'
True
>>> 300 in b'pybee'  # should throw ValueError because 300 is not a valid byte
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: byte must be in range(0, 256)
```